src/js/subtraction/components/__tests__/SubtractionList.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { createBrowserHistory } from "history";
+import nock from "nock";
+import React from "react";
+import { describe, expect, it, vi } from "vitest";
+import { createFakeAccount, mockGetAccountAPI } from "../../../../tests/fake/account";
+import { renderWithRouter } from "../../../../tests/setupTests";
+import { AdministratorRoles } from "../../../administration/types";
+import SubtractionList from "../SubtractionList";
+import SubtractionToolbar from "../SubtractionToolbar";
+
+const subtractions = {
+    documents: [{ id: "foo", name: "bar", user: { handle: "test" } }],
+    total_count: 1,
+};
```

Comment:
If your feeling up to it please create a new faker instead. Use the existing ones for guidance. In general we want fakers for all data that will be returned from the API